extensions/interactions/base_test.py

```diff
@@ -434,6 +435,13 @@ def test_default_interactions_are_valid(self):
             except Exception:
                 pass
 
+            try:
+                self.assertTrue(os.path.isfile(os.path.join(
+                    interaction_dir, '%s_test.py' % interaction_id)))
+                interaction_dir_optional_dirs_and_files_count += 1
+            except Exception:
+                pass
```

Comment:
Why it this a try block?

Reply:
As all the checks are in try block, I am keeping this as well.